src/Jank/TermManagement/ui.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+export const Button = styled.button`
+  margin: 10px;
+  padding: 5px 10px;
+  background-color: white;
+  border-radius: 5px;
+  border: 1px solid black;
+`;
+
+export const DeleteButton = styled(Button)`
+  color: red;
+`;
+
+export const AddButton = styled(Button)`
+  color: green;
+  font-size: 20px;
+`;
+
+export const TermText = styled.span`
+  margin-top: 10px;
+  font-size: 20px;
+  text-transform: capitalize;
+`;
+
+export const TermInput = styled.input`
+  font-size: 18px;
+  margin: 5px;
+`;
+
+export const AddLabel = styled.label`
+  padding: 5px;
+  font-size: 20px;
```

Comment:
We are using the same `font-size` in almost all components here - maybe it makes sense to define this on some parent component instead? If we don't have a wrapping component yet, maybe it makes sense to introduce one and define it there?